package/extra-dts/BB-LCD-ILI9488-ADS7846.dts

```diff
@@ -6,8 +6,26 @@
     part-number = "ZOLA-LCD-ILI9488-ADS7846";
     version = "00A1";
 
-    /* Fragment 1: Pinmux: Separate SPI and GPIO Pins */
+
+    /* Fragment 0: Release J31 pins from legacy OCP/Helper ownership */
     fragment@0 {
+        target = <&ocp>;
+        __overlay__ {
```

Comment:
- We should add comment to say make it clear to readers that these are ping mappings for V2 commcard.
- We should also think about how this affects when image is running ov V1. For example A13 has changed to be MMC0_CD to SPI_CLK. How will this affect MMC0.